experimental/tau_value_determination.py

```diff
@@ -163,7 +208,7 @@ def draw_l_shape( l_shape, best_tau, x_value, y_value, channel, variable ):
         plt.savefig( 'plots/tau_from_L_shape_%s_channel_%s_MC.png' % ( channel, variable ) )
 
 def get_data_histogram( channel, variable, met_type ):
-    fit_result_input = '../../data/8TeV/%(variable)s/fit_results/central/fit_results_%(channel)s_%(met_type)s.txt'
+    fit_result_input = 'data/absolute_eta_M3_angle_bl/8TeV/%(variable)s/fit_results/central/fit_results_%(channel)s_%(met_type)s.txt'
```

Comment:
Maybe add option to use fit results from different combinations of fit variables?